### PR TITLE
Edits for the chemistry module

### PR DESCRIPTION
**Pull request by nborsato**
I just went through the chemistry example running the code and reading through the docs, I've made some edits to both the code and the text to try to help clarify some things.

### CODE REVIEW

**Review by Hoeijmakers — APPROVED** [review]
I think this looks good. Not sure why line 22 is needed, but it's not a big deal. Good to plot the TP profile.

File: docs/shone/examples/chemistry.rst [hunk 1]
```diff
@@ -16,9 +16,18 @@ Let's construct a atmospheric structure (temperature-pressure curve):
 .. code-block:: python
 
     import numpy as np
+    import matplotlib.pyplot as plt
 
     pressure = np.geomspace(1e-6, 10, 15)  # [bar]
-    temperature = 2300 * (pressure / 0.1) ** 0.1  # [K]
+    temperature = 2300 * (np.geomspace(1e-6, 10, pressure.size) / 0.1) ** 0.1  # [K]
```
Comment (bmorris3) [inline docs/shone/examples/chemistry.rst:22]: The way I wrote it was redundant. Here's a suggestion for consistency and clarity:
```suggestion
    temperature = 2300 * (pressure / 0.1) ** 0.1  # [K]
```

Comment (bmorris3) [reply docs/shone/examples/chemistry.rst:22]: Bonus points if you make the same correction in your source code on line 38: https://github.com/bmorris3/shone/pull/10/files#diff-01b0298e1f6dacf74ab16c88d2889005257e03744cf3ada15277fcbc254a0aa6R38

File: docs/shone/examples/chemistry.rst [hunk 2]
```diff
@@ -36,9 +45,7 @@ Let's construct a atmospheric structure (temperature-pressure curve):
         ylabel='Pressure [bar]'
     )
 
-Radiative transfer calculations require the mixing ratio of each atmospheric species, which are
-functions of temperature, pressure, atmospheric metallicity, and C/O ratio. We specify these
-parameters like so:
+Radiative transfer calculations require the mixing ratios, which are the proportions of each species that exist within the atmosphere. These ratios are functions of temperature, pressure, atmospheric metallicity, and the C/O ratio. If we provide these value to FastChem we'll obtain the ratios. We specify these parameters as follows:
```
Comment (bmorris3) [inline docs/shone/examples/chemistry.rst:48]:
```suggestion
Radiative transfer calculations require the volume mixing ratio (VMR) of each atmospheric species. The VMR is a function of temperature, pressure, atmospheric metallicity, and the carbon-to-oxygen ratio. We specify these parameters as follows:
```

File: docs/shone/examples/chemistry.rst [hunk 3]
```diff
@@ -86,28 +93,43 @@ Returns a table like this:
           <th>e-</th>
           <td>0</td>
           <td>Electron</td>
-          <td>0.000549</td>
+          <td>0.00054857990907</td>
```
Comment (bmorris3) [inline docs/shone/examples/chemistry.rst:96]:
```suggestion
          <td>0.000549</td>
```

File: docs/shone/examples/chemistry.rst [hunk 4]
```diff
@@ -86,28 +93,43 @@ Returns a table like this:
...
           <td>element</td>
         </tr>
         <tr>
           <th>Al</th>
           <td>1</td>
           <td>Aluminium</td>
-          <td>26.981538</td>
+          <td>26.9815385</td>
```
Comment (bmorris3) [inline docs/shone/examples/chemistry.rst:103]:
```suggestion
          <td>26.98</td>
```

**Review by bmorris3 — CHANGES_REQUESTED** [review]
Thanks @nborsato! Here's some feedback.

**Review by bmorris3 — APPROVED** [review]
Thanks @nborsato!

**Pull request merged**